content/ref/python/log.md

```diff
@@ -120,19 +120,19 @@ For more and more detailed examples, see
 ```python
 import wandb
 
-run = wandb.init()
-run.log({"accuracy": 0.9, "epoch": 5})
+with wandb.init() as run:
+    run.log({"accuracy": 0.9, "epoch": 5})
```

Comment:
Pretty please change each tab stop to 2 spaces. Otherwise, the browser will expand each one to 8 spaces, which is very........wide.  🙏